S2 In-line alert token updates

nabuhasan opened the pull request:
<!--- Title: Provide a general summary of your changes in the Title above -->
<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao, @KayiuCarlson, @nabuhasan -->
<!--- Approval: PRs require two reviews at the minimum (one from the engineering team and one from the design team) in order to be considered "approved" and ready to merge -->

## Description

Added net-new tokens:

- informative-subtle-background-color-default
- positive-subtle-background-color-default
- notice-subtle-background-color-default
- negative-subtle-background-color-default
- neutral-subtle-background-color-default

These color tokens were added into both light and dark sets. 

Placement of tokens into existing tokens organization was determined in this [paper doc](https://paper.dropbox.com/doc/Adding-subtle-backgrounds-to-Tokens-Studio--CNwSF2RY0kwa9srOpzvFCuy5Ag-qeZ1Yqq9Rrj7VvWt38eo9). 

Variables were also created in S2/Variables with these new additions. 

<!--- Describe your changes in detail, including a list of changes -->

## Motivation and context
These changes occurred as a result of scaling things for S2 design language and experiences.
<!--- Why is this change required? What problem does it solve? -->

## Related issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in #spectrum_tokens_talk or design workshop, first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue on the next line: -->

View [Jira task](https://jira.corp.adobe.com/browse/SDS-13356)

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [X] Minor (add a new token, changing a value, deprecating a token; non-breaking change which adds functionality) </li>
<li> [ ] Major (deleting a token, changing token value type, renaming a token by deprecating the old one; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

<li> [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [X] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

mrcjhicks commented:
**[TESTING spectrum2-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8808742333)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/`
TOKENS STUDIO DATA IS COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.neutral-subtle.default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.informative-subtle.default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.positive-subtle.default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.notice-subtle.default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.negative-subtle.default
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (5) neutral-subtle-background-color-default, informative-subtle-background-color-default, positive-subtle-background-color-default, notice-subtle-background-color-default, negative-subtle-background-color-default
```
</p></details>

TEST COMPLETE

nabuhasan commented on src/tokens-studio/spectrum2-colors/$themes.json:
```diff
@@ -36,6 +37,12 @@
       "Icon.red": "1f734c4352da33631259a822a4377c36d9129856",
       "Icon.yellow": "34ea0effbfa548c92ac7a28428625b1fc9dbc47e",
       "Alias.overlay": "fa3f19d79b2633f5b0913c442831b94c56bfbd66",
+      "Alias.🚫 drop-shadow": "0e6900e9ce8ddf2c72e312dcc202679dd76f2516",
+      "Alias.drop-shadow.100": "439d299ee483470b504ec8b21ac2309f82da7849",
+      "Alias.drop-shadow.200": "6a04bd46ee13367ed8477c79bb1be3c4e5149748",
+      "Alias.drop-shadow.emphasized-default": "6845015b6692e43f3375947e20d74951d662ac28",
+      "Alias.drop-shadow.emphasized-hover": "e080d6090b53c910bca628a47dda47630c8ba759",
+      "Alias.drop-shadow.elevated": "7ffc61455bb815b6c1129d3d6ccfa224245fbfc0",
```
I didn't modify these, are these due to other updates that I pulled into my branch? @lynnhao

lynnhao replied:
I'm not sure, these updates were [merged](https://github.com/adobe/spectrum-tokens-studio-data/pull/84) three weeks ago 🤔

lynnhao replied:
@karstens @nabuhasan yeah good point. I'm fine with either way. In the S2/Desktop file when we deprecate a component we usually do include a space, but whatever works best for tokens works for me!

karstens replied:
@lynnhao @nabuhasan I am also fine with either way, but I think, *with* space is a little bit more readable in the Figma UI. Lets go *with* space. 👍

lynnhao commented on src/tokens-studio/spectrum2-colors/$themes.json:
```diff
@@ -16,6 +16,7 @@
       "Component.color-area.border": "34a0ec3c7e0acfc890c6ac1021b0fe841c934d32",
       "Component.color-handle.inner-border": "3bdc6483aefb8301420b9ba71fe8172e966ce4f7",
       "Component.color-handle.outer-border": "e7cda9a602a097be1a3a44ea283144c54ab7b405",
+      "Component.color-loupe.🚫drop-shadow": "7a15aeef80d9cc9623427a57fb2e7d9533cc11d6",
```
This was also merged I think @nabuhasan @GarthDB

lynnhao commented:
@nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1. 

For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. 

<img width="176" alt="Screenshot 2024-04-23 at 5 24 33 PM" src="https://github.com/adobe/spectrum-tokens-studio-data/assets/16598521/e2376754-71e0-47a9-88be-f37e33460a92">
<img width="187" alt="Screenshot 2024-04-23 at 5 24 12 PM" src="https://github.com/adobe/spectrum-tokens-studio-data/assets/16598521/62c58565-6f60-4d9b-a8cd-6cb27feef035">

nabuhasan commented:
> @nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1.
> 
> For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. At least, that is how I was interpreting the paper doc options!
> 
> <img alt="Screenshot 2024-04-23 at 5 24 33 PM" width="176" src="https://private-user-images.githubusercontent.com/16598521/325030438-e2376754-71e0-47a9-88be-f37e33460a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0MzgtZTIzNzY3NTQtNzFlMC00N2E5LTg4YmUtZjM3ZTMzNDYwYTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNzE3NzNjZTA1MGZjZDZhYzc3N2Y2ZWEwMjY1ZjE3ZWQ5NDQ5OGMwMjE1MzA1YTQwZjQyODAxOTVjNzM0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvYvYQpa8SZD-mjq6Oq1Ij2fHbuKPh1k3HmVvHTdpv4"> <img alt="Screenshot 2024-04-23 at 5 24 12 PM" width="187" src="https://private-user-images.githubusercontent.com/16598521/325030442-62c58565-6f60-4d9b-a8cd-6cb27feef035.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0NDItNjJjNTg1NjUtNmY2MC00ZDliLWE4Y2QtNmNiMjdmZWVmMDM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNTNlNWZjZjVjN2I2MDQyODViZTE3MTRlZWRkMDhhMGI1NDljMzM0MmY1MDY3M2M1NzEzYTdlNDA3NjYxYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZaM-qKaQF5nUmRINRbyQ4u_fxQC5Fe6qxOnUUaKKc">

Ah yeah I can see how that's confusing @lynnhao. So I noticed that neutral selected is technically it's own group in our organization and doesn't fall under just neutral. I originally had it written out that it falls under neutral. Will adjust! 

<img width="804" alt="Screenshot 2024-04-23 at 9 00 16 PM" src="https://github.com/adobe/spectrum-tokens-studio-data/assets/129990790/438b2029-2cd7-41cc-bdb4-506df868ba05">

mrcjhicks commented:
**[TESTING spectrum2-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8810911884)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/`
TOKENS STUDIO DATA IS COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.neutral.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.informative.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.positive.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.notice.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.negative.subtle-default
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (5) neutral-subtle-background-color-default, informative-subtle-background-color-default, positive-subtle-background-color-default, notice-subtle-background-color-default, negative-subtle-background-color-default
```
</p></details>

TEST COMPLETE

nabuhasan commented:
> > > @nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1.
> > > For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. At least, that is how I was interpreting the paper doc options!
> > > <img alt="Screenshot 2024-04-23 at 5 24 33 PM" width="176" src="https://private-user-images.githubusercontent.com/16598521/325030438-e2376754-71e0-47a9-88be-f37e33460a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0MzgtZTIzNzY3NTQtNzFlMC00N2E5LTg4YmUtZjM3ZTMzNDYwYTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNzE3NzNjZTA1MGZjZDZhYzc3N2Y2ZWEwMjY1ZjE3ZWQ5NDQ5OGMwMjE1MzA1YTQwZjQyODAxOTVjNzM0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvYvYQpa8SZD-mjq6Oq1Ij2fHbuKPh1k3HmVvHTdpv4"> <img alt="Screenshot 2024-04-23 at 5 24 12 PM" width="187" src="https://private-user-images.githubusercontent.com/16598521/325030442-62c58565-6f60-4d9b-a8cd-6cb27feef035.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0NDItNjJjNTg1NjUtNmY2MC00ZDliLWE4Y2QtNmNiMjdmZWVmMDM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNTNlNWZjZjVjN2I2MDQyODViZTE3MTRlZWRkMDhhMGI1NDljMzM0MmY1MDY3M2M1NzEzYTdlNDA3NjYxYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZaM-qKaQF5nUmRINRbyQ4u_fxQC5Fe6qxOnUUaKKc">
> > 
> > 
> > Ah yeah I can see how that's confusing @lynnhao. So I noticed that neutral selected is technically it's own group in our organization and doesn't fall under just neutral. I originally had it written out that it falls under neutral. Will adjust!
> > <img alt="Screenshot 2024-04-23 at 9 00 16 PM" width="804" src="https://private-user-images.githubusercontent.com/129990790/325075040-438b2029-2cd7-41cc-bdb4-506df868ba05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzUyMDIsIm5iZiI6MTcxMzk3NDkwMiwicGF0aCI6Ii8xMjk5OTA3OTAvMzI1MDc1MDQwLTQzOGIyMDI5LTJjZDctNDFjYy1iZGI0LTUwNmRmODY4YmEwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjA4MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDBiMmU0ZmFmYjcwOGU1ZjIyZTkwNmFlNjE0OWY2MjIwYzVjYWRhN2JmYTBkMGYyNTZhYmUyMmJmNWNjNGRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BSDn-xj01jyR8DKkoh1rLyHWprM90fEpmlvap8lQvk">
> 
> Thank you, this makes sense to me now! One last thing, should we also re-order the tokens? In S2/Variables, they show up like this: <img alt="Screenshot 2024-04-24 at 11 07 49 AM" width="608" src="https://private-user-images.githubusercontent.com/16598521/325335074-736ddef9-b106-4fdf-beb0-072d6defa985.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5ODQwMDAsIm5iZiI6MTcxMzk4MzcwMCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUzMzUwNzQtNzM2ZGRlZjktYjEwNi00ZmRmLWJlYjAtMDcyZDZkZWZhOTg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDE4MzUwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ODAyYTg4NDhhZWMzMmVlZGQ5YjRjZjFmOTA1N2QzZDVhODNhMmNlZjI5ZTI3NzZhZDJlYWQ3Y2ExMTkxM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qX9sQfLsB_jCePdzFhCQJEgixpvNjCAcVC1ZgeB4luI">
> 
> I think in the paper doc, you had them at the end (after `key-focus`).

Oh I changed the order of them in S2/Variables. I thought it might be missed if it came after key-focus states. @lynnhao I can see it working either way so I don't have a strong a preference.

lynnhao commented:
> > > > @nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1.
> > > > For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. At least, that is how I was interpreting the paper doc options!
> > > > <img alt="Screenshot 2024-04-23 at 5 24 33 PM" width="176" src="https://private-user-images.githubusercontent.com/16598521/325030438-e2376754-71e0-47a9-88be-f37e33460a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0MzgtZTIzNzY3NTQtNzFlMC00N2E5LTg4YmUtZjM3ZTMzNDYwYTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNzE3NzNjZTA1MGZjZDZhYzc3N2Y2ZWEwMjY1ZjE3ZWQ5NDQ5OGMwMjE1MzA1YTQwZjQyODAxOTVjNzM0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvYvYQpa8SZD-mjq6Oq1Ij2fHbuKPh1k3HmVvHTdpv4"> <img alt="Screenshot 2024-04-23 at 5 24 12 PM" width="187" src="https://private-user-images.githubusercontent.com/16598521/325030442-62c58565-6f60-4d9b-a8cd-6cb27feef035.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0NDItNjJjNTg1NjUtNmY2MC00ZDliLWE4Y2QtNmNiMjdmZWVmMDM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNTNlNWZjZjVjN2I2MDQyODViZTE3MTRlZWRkMDhhMGI1NDljMzM0MmY1MDY3M2M1NzEzYTdlNDA3NjYxYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZaM-qKaQF5nUmRINRbyQ4u_fxQC5Fe6qxOnUUaKKc">
> > > 
> > > 
> > > Ah yeah I can see how that's confusing @lynnhao. So I noticed that neutral selected is technically it's own group in our organization and doesn't fall under just neutral. I originally had it written out that it falls under neutral. Will adjust!
> > > <img alt="Screenshot 2024-04-23 at 9 00 16 PM" width="804" src="https://private-user-images.githubusercontent.com/129990790/325075040-438b2029-2cd7-41cc-bdb4-506df868ba05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzUyMDIsIm5iZiI6MTcxMzk3NDkwMiwicGF0aCI6Ii8xMjk5OTA3OTAvMzI1MDc1MDQwLTQzOGIyMDI5LTJjZDctNDFjYy1iZGI0LTUwNmRmODY4YmEwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjA4MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDBiMmU0ZmFmYjcwOGU1ZjIyZTkwNmFlNjE0OWY2MjIwYzVjYWRhN2JmYTBkMGYyNTZhYmUyMmJmNWNjNGRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BSDn-xj01jyR8DKkoh1rLyHWprM90fEpmlvap8lQvk">
> > 
> > 
> > Thank you, this makes sense to me now! One last thing, should we also re-order the tokens? In S2/Variables, they show up like this: <img alt="Screenshot 2024-04-24 at 11 07 49 AM" width="608" src="https://private-user-images.githubusercontent.com/16598521/325335074-736ddef9-b106-4fdf-beb0-072d6defa985.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5ODQwMDAsIm5iZiI6MTcxMzk4MzcwMCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUzMzUwNzQtNzM2ZGRlZjktYjEwNi00ZmRmLWJlYjAtMDcyZDZkZWZhOTg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDE4MzUwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ODAyYTg4NDhhZWMzMmVlZGQ5YjRjZjFmOTA1N2QzZDVhODNhMmNlZjI5ZTI3NzZhZDJlYWQ3Y2ExMTkxM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qX9sQfLsB_jCePdzFhCQJEgixpvNjCAcVC1ZgeB4luI">
> > I think in the paper doc, you had them at the end (after `key-focus`).
> 
> Oh I changed the order of them in S2/Variables. I thought it might be missed if it came after key-focus states. @lynnhao I can see it working either way so I don't have a strong a preference.

Ooh I see! I feel putting it at the end might be clearer, since the other states (hover, down, etc) don't apply to it. The way it's organized now seems like it's interchangeable for `default` even though it's not.

nabuhasan commented:
> > > > > @nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1.
> > > > > For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. At least, that is how I was interpreting the paper doc options!
> > > > > <img alt="Screenshot 2024-04-23 at 5 24 33 PM" width="176" src="https://private-user-images.githubusercontent.com/16598521/325030438-e2376754-71e0-47a9-88be-f37e33460a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0MzgtZTIzNzY3NTQtNzFlMC00N2E5LTg4YmUtZjM3ZTMzNDYwYTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNzE3NzNjZTA1MGZjZDZhYzc3N2Y2ZWEwMjY1ZjE3ZWQ5NDQ5OGMwMjE1MzA1YTQwZjQyODAxOTVjNzM0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvYvYQpa8SZD-mjq6Oq1Ij2fHbuKPh1k3HmVvHTdpv4"> <img alt="Screenshot 2024-04-23 at 5 24 12 PM" width="187" src="https://private-user-images.githubusercontent.com/16598521/325030442-62c58565-6f60-4d9b-a8cd-6cb27feef035.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0NDItNjJjNTg1NjUtNmY2MC00ZDliLWE4Y2QtNmNiMjdmZWVmMDM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNTNlNWZjZjVjN2I2MDQyODViZTE3MTRlZWRkMDhhMGI1NDljMzM0MmY1MDY3M2M1NzEzYTdlNDA3NjYxYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZaM-qKaQF5nUmRINRbyQ4u_fxQC5Fe6qxOnUUaKKc">
> > > > 
> > > > 
> > > > Ah yeah I can see how that's confusing @lynnhao. So I noticed that neutral selected is technically it's own group in our organization and doesn't fall under just neutral. I originally had it written out that it falls under neutral. Will adjust!
> > > > <img alt="Screenshot 2024-04-23 at 9 00 16 PM" width="804" src="https://private-user-images.githubusercontent.com/129990790/325075040-438b2029-2cd7-41cc-bdb4-506df868ba05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzUyMDIsIm5iZiI6MTcxMzk3NDkwMiwicGF0aCI6Ii8xMjk5OTA3OTAvMzI1MDc1MDQwLTQzOGIyMDI5LTJjZDctNDFjYy1iZGI0LTUwNmRmODY4YmEwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjA4MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDBiMmU0ZmFmYjcwOGU1ZjIyZTkwNmFlNjE0OWY2MjIwYzVjYWRhN2JmYTBkMGYyNTZhYmUyMmJmNWNjNGRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BSDn-xj01jyR8DKkoh1rLyHWprM90fEpmlvap8lQvk">
> > > 
> > > 
> > > Thank you, this makes sense to me now! One last thing, should we also re-order the tokens? In S2/Variables, they show up like this: <img alt="Screenshot 2024-04-24 at 11 07 49 AM" width="608" src="https://private-user-images.githubusercontent.com/16598521/325335074-736ddef9-b106-4fdf-beb0-072d6defa985.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5ODQwMDAsIm5iZiI6MTcxMzk4MzcwMCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUzMzUwNzQtNzM2ZGRlZjktYjEwNi00ZmRmLWJlYjAtMDcyZDZkZWZhOTg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDE4MzUwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ODAyYTg4NDhhZWMzMmVlZGQ5YjRjZjFmOTA1N2QzZDVhODNhMmNlZjI5ZTI3NzZhZDJlYWQ3Y2ExMTkxM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qX9sQfLsB_jCePdzFhCQJEgixpvNjCAcVC1ZgeB4luI">
> > > I think in the paper doc, you had them at the end (after `key-focus`).
> > 
> > 
> > Oh I changed the order of them in S2/Variables. I thought it might be missed if it came after key-focus states. @lynnhao I can see it working either way so I don't have a strong a preference.
> 
> Ooh I see! I feel putting it at the end might be clearer, since the other states (hover, down, etc) don't apply to it. The way it's organized now seems like it's interchangeable for `default` even though it's not.

@lynnhao sounds good, updated order in S2/Variables.

GarthDB reviewed (approved):
If @karstens is good with the weird changes at the top, I'm good with the rest.

lynnhao commented:
> > > > > > @nabuhasan I'm a little confused since in the S2/Variables library, the structure here feels more similar to Option 1.5 from the paper doc, not Option 1.
> > > > > > For example, `informative-subtle` is a separate section from `informative`, instead of being underneath it, ie. `informative/informative-subtle`. At least, that is how I was interpreting the paper doc options!
> > > > > > <img alt="Screenshot 2024-04-23 at 5 24 33 PM" width="176" src="https://private-user-images.githubusercontent.com/16598521/325030438-e2376754-71e0-47a9-88be-f37e33460a92.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0MzgtZTIzNzY3NTQtNzFlMC00N2E5LTg4YmUtZjM3ZTMzNDYwYTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNzE3NzNjZTA1MGZjZDZhYzc3N2Y2ZWEwMjY1ZjE3ZWQ5NDQ5OGMwMjE1MzA1YTQwZjQyODAxOTVjNzM0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvYvYQpa8SZD-mjq6Oq1Ij2fHbuKPh1k3HmVvHTdpv4"> <img alt="Screenshot 2024-04-23 at 5 24 12 PM" width="187" src="https://private-user-images.githubusercontent.com/16598521/325030442-62c58565-6f60-4d9b-a8cd-6cb27feef035.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzExNTAsIm5iZiI6MTcxMzkzMDg1MCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUwMzA0NDItNjJjNTg1NjUtNmY2MC00ZDliLWE4Y2QtNmNiMjdmZWVmMDM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDAzNTQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNTNlNWZjZjVjN2I2MDQyODViZTE3MTRlZWRkMDhhMGI1NDljMzM0MmY1MDY3M2M1NzEzYTdlNDA3NjYxYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1IZaM-qKaQF5nUmRINRbyQ4u_fxQC5Fe6qxOnUUaKKc">
> > > > > 
> > > > > 
> > > > > Ah yeah I can see how that's confusing @lynnhao. So I noticed that neutral selected is technically it's own group in our organization and doesn't fall under just neutral. I originally had it written out that it falls under neutral. Will adjust!
> > > > > <img alt="Screenshot 2024-04-23 at 9 00 16 PM" width="804" src="https://private-user-images.githubusercontent.com/129990790/325075040-438b2029-2cd7-41cc-bdb4-506df868ba05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzUyMDIsIm5iZiI6MTcxMzk3NDkwMiwicGF0aCI6Ii8xMjk5OTA3OTAvMzI1MDc1MDQwLTQzOGIyMDI5LTJjZDctNDFjYy1iZGI0LTUwNmRmODY4YmEwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjA4MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDBiMmU0ZmFmYjcwOGU1ZjIyZTkwNmFlNjE0OWY2MjIwYzVjYWRhN2JmYTBkMGYyNTZhYmUyMmJmNWNjNGRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BSDn-xj01jyR8DKkoh1rLyHWprM90fEpmlvap8lQvk">
> > > > 
> > > > 
> > > > Thank you, this makes sense to me now! One last thing, should we also re-order the tokens? In S2/Variables, they show up like this: <img alt="Screenshot 2024-04-24 at 11 07 49 AM" width="608" src="https://private-user-images.githubusercontent.com/16598521/325335074-736ddef9-b106-4fdf-beb0-072d6defa985.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5ODQwMDAsIm5iZiI6MTcxMzk4MzcwMCwicGF0aCI6Ii8xNjU5ODUyMS8zMjUzMzUwNzQtNzM2ZGRlZjktYjEwNi00ZmRmLWJlYjAtMDcyZDZkZWZhOTg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDE4MzUwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1ODAyYTg4NDhhZWMzMmVlZGQ5YjRjZjFmOTA1N2QzZDVhODNhMmNlZjI5ZTI3NzZhZDJlYWQ3Y2ExMTkxM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qX9sQfLsB_jCePdzFhCQJEgixpvNjCAcVC1ZgeB4luI">
> > > > I think in the paper doc, you had them at the end (after `key-focus`).
> > > 
> > > 
> > > Oh I changed the order of them in S2/Variables. I thought it might be missed if it came after key-focus states. @lynnhao I can see it working either way so I don't have a strong a preference.
> > 
> > 
> > Ooh I see! I feel putting it at the end might be clearer, since the other states (hover, down, etc) don't apply to it. The way it's organized now seems like it's interchangeable for `default` even though it's not.
> 
> @lynnhao sounds good, updated order in S2/Variables.

Thank you! I noticed `subtle-informative` was still in the previous spot so I reorganized that one manually. Let me know if that looks ok. Everything else looks good to me!

karstens commented:
> If @karstens is good with the weird changes at the top, I'm good with the rest.

@GarthDB yes, as expected the "weird changes" appear in the other new PRs, too. So, I will merge this and in case there are  merge conflicts in the other newer PRs, I will fix them.

mrcjhicks commented:
**[SYNCING spectrum2-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8833266665)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/`
TOKENS STUDIO DATA IS COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.neutral.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.informative.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.positive.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.notice.subtle-default
UPDATED METADATA - SPECTRUM UUID: spectrum2/alias.Alias.background.semantic.negative.subtle-default
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (5) neutral-subtle-background-color-default, informative-subtle-background-color-default, positive-subtle-background-color-default, notice-subtle-background-color-default, negative-subtle-background-color-default
```
</p></details>

WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/spectrum2/alias/dark.json`
WROTE: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/spectrum2/alias/light.json`
WROTE: `/spectrum-tokens/packages/tokens/src/semantic-color-palette.json`
WROTE: `/spectrum-tokens/packages/tokens/src/color-aliases.json`

SYNC COMPLETE
[Tokens Studio Pull Request](https://github.com/adobe/spectrum-tokens-studio-data/pull/118)

[Spectrum Tokens Pull Request](https://github.com/adobe/spectrum-tokens/pull/325)
